test/backend_sum.in.cpp

```diff
@@ -427,3 +427,30 @@ NGRAPH_TEST(${BACKEND_NAME}, sum_5d_to_scalar)
     backend->call_with_validate(f, {result}, {a});
     EXPECT_EQ(std::vector<float>{243.}, read_vector<float>(result));
 }
+
+NGRAPH_TEST(${BACKEND_NAME}, sum_stable_acc)
+{
+    Shape shape_a{10, 10, 10, 30};
+    auto A = make_shared<op::Parameter>(element::f32, shape_a);
+
+    Shape shape_rt{10};
+    auto f =
+        make_shared<Function>(make_shared<op::Sum>(A, AxisSet{1, 2, 3}), op::ParameterVector{A});
+
+    test::Uniform<float> rng(1000.0f, 1000.1f, 2112);
+    vector<vector<float>> args;
+    for (shared_ptr<op::Parameter> param : f->get_parameters())
+    {
+        vector<float> tensor_val(shape_size(param->get_shape()));
+        rng.initialize(tensor_val);
+        args.push_back(tensor_val);
+    }
+
+    auto cpu_func = clone_function(*f);
+    auto bk_func = clone_function(*f);
+
+    auto cpu_results = execute(cpu_func, args, "CPU");
```

Comment:
why are we comparing to CPU and not interpreter?

Reply:
INTERPRETER is also producing incorrect results. The purpose of this test is address the issue of failures in tensorflow tests. Both GPU and INTERPRETER produced similar results which differs from tensorflow. CPU  passes the tf tests.

Reply:
@AyanmoI, if it is easy, we could quickly change the INTERPRETER to use the kahan sum and try comparing against it. Can you take a look?

Reply:
The current situation with respect to `Sum` is a bit messy. `INTERPRETER` is a naive sum, while `CPU` was switched to a Kahan sum some time ago. The behavior should not be inconsistent like this; if it's part of the spec that `Sum` needs to be (or for that matter is _allowed_ to be) a stable sum, we should write that into the spec.

@rkimballn1 has raised this issue a number of times, but we haven't gotten around to resolving it. I'd actually favor separate `Sum` and `StableSum` ops, and leaving it up to the bridge to select the correct one.

Reply:
Also plausible: the current `Sum`'s spec could be updated with some kind of stability guarantee, and we could add a `NaiveSum` if anybody actually wants it.

Reply:
I just pushed a kahan sum implementation for INTERPRETER `Sum`

Reply:
Also @jbobba , changed to comparing with INTERPRETER rather than CPU

Reply:
Please suggest if there needs to be two separate `sum`s for INTERPRETER